syft/format/common/spdxhelpers/to_format_model_test.go

```diff
@@ -777,3 +778,74 @@ func Test_toSPDXID(t *testing.T) {
 		})
 	}
 }
+
+func Test_otherLicenses(t *testing.T) {
```

Comment:
Nice - thanks for adding this!

Reply:
this was a migration from elsewhere to help with package imports (the test utility is using a top-level format function, so was causing a cycle)